Client/Frontend/Library/HistoryPanel/HistoryPanelWithGroups.swift

```diff
@@ -323,14 +324,21 @@ class HistoryPanelWithGroups: UIViewController, LibraryPanel, Loggable, Notifica
         cell.descriptionLabel.isHidden = false
         cell.leftImageView.layer.borderColor = HistoryPanelUX.IconBorderColor.cgColor
         cell.leftImageView.layer.borderWidth = HistoryPanelUX.IconBorderWidth
-        cell.leftImageView.contentMode = .center
-        cell.leftImageView.setImageAndBackground(forIcon: site.icon, website: site.tileURL) { [weak cell] in
-            cell?.leftImageView.image = cell?.leftImageView.image?.createScaled(CGSize(width: HistoryPanelUX.IconSize, height: HistoryPanelUX.IconSize))
+        cell.leftImageView.contentMode = .scaleAspectFit
+        getFavIcon(for: site) { [weak cell] image in
+            cell?.leftImageView.image = image
+            cell?.leftImageView.backgroundColor = UIColor.theme.general.faviconBackground
         }
-        
+
         return cell
     }
     
+    private func getFavIcon(for site: Site, completion: @escaping (UIImage?) -> Void) {
+        siteImageHelper.fetchImageFor(site: site, imageType: .favicon, shouldFallback: false) { image in
```

Comment:
wondering if fallback does anything 🤔

Reply:
if I understood correctly if `shouldFallback` is true fallback to the hero image but the usage we have for FavIcon `shouldFallback` is always false